src/main/resources/category/java/common.xml

```diff
@@ -2209,7 +2209,7 @@ class ValueGood {
     ((.//UnaryExpression[@Operator=('++', '--')]|.//InfixExpression[@Operator=('+', '-')]/NumericLiteral[@ValueAsInt=1]|.//AssignmentExpression[@Operator=('+=', '-=')]/NumericLiteral[@ValueAsInt=1])
```

Comment:
The rule "message" of `AvoidInfiniteRecursion` is empty, this breaks loading the plugin in SonarQube.

Reply:
good point, was forgotten